Resources/algorithm-results/runtime-statistics-table.php

```diff
@@ -1,64 +1,53 @@
+<p>The following table describes the default runtime statistics:</p>
 <?php
-
-$getRuntimeStatisticsTable = function($isLiveMode) {
-
-    echo "<p>The following table describes the default runtime statistics:</p>";
-
     include(DOCS_RESOURCES."/glossary.php");
-    echo "
-    <table class='qc-table table'>
-      <thead>
-        <tr>
-          <th style='width: 25%'>Statistic</th>
-          <th style='width: 75%'>Description</th>
-        </tr>
-      </thead>
-      <tbody>
-      ";
-    
-    if (!$isLiveMode) 
-    {
-        echo "
-        <tr>
-          <td>Capacity</td>
-          <td>{$definitionByTerm['capacity']}</td>
-        </tr>"; 
-    }
```

Comment:
@oussamanahdi Table logic is inverted to the conversion.

 - All KPIs on both +++ 1 additional KPI in live.